server/frontend/src/components/Buckets/View.vue

```diff
@@ -351,22 +351,9 @@ export default {
         ["query_format", "advanced"],
         ["resolution", "---"],
         ["j_top", "OR"],
-        ["f1", "OP"],
-        ["o2", "equals"],
-        ["f2", "product"],
-        ["v2", "Web Compatibility"],
-        ["o3", "equals"],
-        ["f3", "component"],
-        ["v3", "Site Reports"],
-        ["f4", "CP"],
-        ["f5", "OP"],
-        ["o6", "equals"],
-        ["f6", "product"],
-        ["v6", "Web Compatibility"],
-        ["o7", "equals"],
-        ["f7", "component"],
-        ["v7", "Privacy: Site Reports"],
-        ["f8", "CP"],
+        ["o1", "substring"],
```

Comment:
I think you should also search in Web Compatibility :: [Privacy] Site Report components, otherwise bugs won't show up until they have been triaged.

Reply:
Most of the bugs in there were created by SV, so they all have the keyword, so I ignored that. But you're right, there's no reason not to include them. Instead of replacing the entire query, I just added another matching row to the "any" group, so it now looks like this:

![Screenshot 2025-04-24 at 16 11 23](https://github.com/user-attachments/assets/8581fbc1-b5d9-4002-8dfb-95849d2828d5)